guillotina/api/search.py

```diff
@@ -116,7 +117,7 @@ def __init__(self, context, request, security=False):
 
 
 @configure.service(
-    context=IResource, method='POST',
+    context=IContainer, method='POST',
```

Comment:
Why are we removing this feature, its good to be able to delete a catalog on a subresource

Reply:
Do we? I don't understand why we would want this?

Reply:
Because the resource could be a subindex

Reply:
alright, I guess